go/vt/vttablet/grpctabletconn/conn.go

```diff
@@ -863,6 +935,74 @@ func (conn *gRPCQueryClient) ReserveExecute(ctx context.Context, target *querypb
 	return sqltypes.Proto3ToResult(reply.Result), reply.ReservedId, conn.tablet.Alias, nil
 }
 
+//ReserveStreamExecute implements the queryservice interface
+func (conn *gRPCQueryClient) ReserveStreamExecute(ctx context.Context, target *querypb.Target, preQueries []string, sql string, bindVariables map[string]*querypb.BindVariable, transactionID int64, options *querypb.ExecuteOptions, callback func(*sqltypes.Result) error) (reservedID int64, alias *topodatapb.TabletAlias, err error) {
```

Comment:
there is a lot of copy and paste with the code block above :point_up: 

any chance we could extract common code so we don't have to duplicate it?

Reply:
the code looks very different to me, not sure how to extract it into common code and would it give any better readability.

Reply:
Ok, not easy to do without generics. one of these days...